serial_communication/web/routes/script_inspector.py

```diff
@@ -0,0 +1,61 @@
+"""This script continously ping running scripts to check if they are working well"""
+import time
+import random
+import subprocess
+import os
+from dotenv import load_dotenv
+import requests
+from serial_handler import exception_logger
```

Comment:
What if script is not working then can you import its functionality, please so let me know if it is still creating logs or you better separate logs functionality from flask project or the better option is #84

Reply:
You are right i haven't tested it that way, so the scenario is when first time orange pi restart script it get exception of permission error but not sure this script can rerun that stopped script or not